src/plugins/news/components/create-article-modal/index.js

```diff
@@ -17,10 +17,27 @@ import trimStart from 'lodash-es/trimStart.js';
 const DATED_SLUG_PATTERN = /^\d{4}\/\d{2}\/\d{2}\/[a-z0-9-]+$/;
 const isValidDatedSlug = value => isValidSlug(value, true) && DATED_SLUG_PATTERN.test(trimStart(value));
 
+const parseDatedSlug = (value) => {
+  let ending = '';
+  if (value && (value.endsWith(' ') || value.endsWith('/') || value.endsWith('-'))) {
+    ending = value[value.length - 1];
+  }
+  const slug = createSlug(value.toLowerCase(), true);
+  return slug ? slug + ending : value;
+};
+
+const slugValidator = (value) => {
```

Comment:
@meghedi nit: these can either both have non-dated names (because we're in the create article modal which will always be dated) or they can both have dated names, but not one each